shoop/default_tax/templates/shoop/default_tax/admin/edit.jinja

```diff
@@ -8,19 +8,65 @@
         <div class="col-md-9">
             <form method="post" id="tax_rule_form">
                 {% csrf_token %}
-                {% call content_block(_("General information"), "fa-info-circle") %}
+                {% call content_block(_("Tax rule information"), "fa-info-circle") %}
                     {{ bs3.field(form.enabled) }}
-                    {{ bs3.field(form.tax) }}
+
+                    <div class="row">
+                      <div class="col-sm-8 col-sm-push-4 col-lg-6 col-lg-push-3">
+                        <h3>{{ _("Matching criteria") }}</h3>
+                        <p>
+                          {% trans %}
+                            The following fields determine when this
```

Comment:
True that about the reordering the text :) My thinking was too linear :)
And yes, I think could be on single line. It's not something you'd visit in code anyways but mostly see in formatted fashion in rendered html. This is not a deal breaker for me though, very minor issue in my eyes :)